includes/discovery/sensors.inc.php

```diff
@@ -37,6 +37,11 @@
     include 'includes/discovery/sensors/state/hp.inc.php';
 }
 
+if ($device['os'] == 'quanta') {
```

Comment:
You don't need to have this four lines of text. Those files will be pulled in on discovery.